x-pack/test/integrations_manager_api_integration/apis/list.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import ServerMock from 'mock-http-server';
+import { FtrProviderContext } from '../../ftr_provider_context';
```

Comment:
This line is [failing in CI because the file is missing](https://github.com/elastic/kibana/pull/45781/checks?check_run_id=252266585).

Should this be:

```suggestion
import { FtrProviderContext } from './config.ts';
```

or perhaps
```suggestion
import { FtrProviderContext } from '../../functional/ftr_provider_context';
```

Reply:
Yes, that was my mistake.

Most other api integration tests define their own set of `services` and have their own version of this file because of that, but for now I've decided to import the one from the main `test/api_integration` directory, see https://github.com/elastic/kibana/pull/45781/commits/cf7e04fda302106f975c6cc50d63d9073669256e